app/tsconfig.json

```diff
@@ -19,6 +19,11 @@
     "checkJs": false,
     // Help speed up type checking in larger applications
     "incremental": true,
+    "paths": {
+      "@/*": ["./*"],
+      "@services": ["./src/services"],
+      "@services/*": ["./src/services/*"]
+    },
```

Comment:
I'm on the fence about this approach since:

1. It seems like this object would need updated anytime a new `src/` subdirectory is added? That would require awareness of that requirement.
2. There was already support for alias imports via using `src/`, so I'm wondering if saving two extra characters is worth it. There could be room for better documenting the existing `src/` alias though.

Curious to hear more of your thoughts on this, I don't feel _super_ strongly. May be worth decoupling these changes from the Docker changes so this doesn't hold that up from merging.

Reply:
There isn't a need to update the config each time a new subdirectory is created within `/src` since there's a root alias defined `"@/*": ["./*"]`. 

I tend to lean towards being explicit about configs so I've explicitly added `@/services` just as a boilerplate example on how to register various alias as the project grows. This might be a bit opinionated or rooted in previous work. If so, we can remove these entries. 

- `app/src/services/feature-flags/FeatureFlagManager` -> `@featureflags/FeatureFlagManager`
- `app/src/i18n/messages/es-US/` -> `@esMessages`

`import { messages } from "@esMessages"`

Reply:
Ah okay, the inclusion of `@services` had me confused. This may be subjective, but I don't love that format because there are valid non-alias import paths that look like that, like `@storybook/` and `@testing-library/`. I think I'd prefer removing these subdirectory aliases and let project teams determine whether they want those instead.

I don't mind `@/` but if we're saying that's a better approach, then I think the template should consistently use that alias instead of `src/`. In other words, replacing `from "src/...` with `from "@/...`. I wonder if we can enforce that using the `no-restricted-imports` lint rule.